### DISCUSSION TITLE
Juntar várias colunas em uma linha

### DISCUSSION THREAD

**Discussion by jornalistaCarlosOsorio** [sqlitebrowser/sqlitebrowser#3117] [Q&A]
Estou com dificuldade para colocar a coluna "Bairro" em uma mesma linha. A ideia é que na linha bairro possa aparecer todos dos bairros de uma mesma Zona. Exemplo: 

Zona - 4 
Bairro - FLAMENGO, BOTAFOGO, HUMAITA, URCA

![image](https://user-images.githubusercontent.com/76069092/182262719-be2e6699-7b1e-4086-a18b-4a0001005c42.png)

Obrigado,

**Comment by jornalistaCarlosOsorio** [comment]
Alguém poderia me responder. Acredito que não seja difícil, mas não consegui descobrir.

**Reply by chrisjlocke** [reply]
So instead of 'zone 4' you want 'FLAMENGO, BOTAFOGO, HUMAITA, URCA' ?

You need another table, which contains a row with two fields - the ID in one field, and the neighborhoods in the other.

![image](https://user-images.githubusercontent.com/7872218/193915742-59e98fbf-ca34-4d1c-8636-a718f1c1b6c7.png)

Then add the details in that record.

![image](https://user-images.githubusercontent.com/7872218/193919905-463ac96b-219a-4c97-a399-b6eb63f15efa.png)

If this is what you want, then I can then (try) to explain the JOIN command.

**Reply by jornalistaCarlosOsorio** [reply]
Sim, é exatamente isso. Como posso utilizar JOIN?

**Reply by jornalistaCarlosOsorio** [reply]
Tenho uma tabela com um id e várias linhas com variáveis. Quero juntar essas variáveis em apenas uma linha. Com isso, ficaria no quadro de sua proposta. 
Você pode me ajudar em uma fórmula de JOIN para esse caso?

**Reply by chrisjlocke** [reply]
Can you paste the 'create table' command of the table, and give an example of the record you want to see output?
